src/cephadm/cephadm

```diff
@@ -2597,6 +2605,10 @@ class CephContainer:
             '--rm',
             '--ipc=host',
         ]
+
+        if 'podman' in container_path and os.path.exists('/var/lib/ceph/%s/podman-auth.json' % self.fsid):
+            cmd_args.append('--authfile=/var/lib/ceph/%s/podman-auth.json' % self.fsid)
```

Comment:
Ok. as I understand now, we're now creating some boostrap issue for non-standard use cases:

1. user wants to pull the ceph image to do some things before calling bootstrap, (maybe he wants to verify the ceph version. I don't know)
2. user calls boostrap. 

If we now add the cluster's fsid in here, we end up with:

1. user needs to call `cephadm login ...`
2. then `cephadm pull`
3. `cephadm bootstrap ...` 

Which is not going to work, cause we now have the cluster FSID being required to call pull things. 

I understand the desire to have different logins for different ceph clusters, but we#re creating some some bad UX here. And I'm not sure we want that. 

Or did I miss anything?